src/lib/features/dependent-features/dependent-features-store.ts

```diff
@@ -38,6 +38,23 @@ export class DependentFeaturesStore implements IDependentFeaturesStore {
         return rows.map((row) => row.child);
     }
 
+    async getParentOptions(child: string): Promise<string[]> {
```

Comment:
algorithms:
* select your feature's project
* select all features from this project that are not children themselves

Reply:
Is the naming deliberate? What are the options? getPossibleParentFeatures?

Reply:
yup, I was thinking that parent options is a more elegant way of saying possible parent features :)